package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "django-react-template",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@babel/core": "^7.12.10",
+    "@babel/plugin-transform-runtime": "^7.12.10",
+    "@babel/runtime": "^7.12.5",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "babel": "^6.23.0",
+    "babel-loader": "^8.1.0",
+    "babel-preset-proposals": "^0.3.0",
+    "django-react-loader": "^0.1.4",
+    "react": "^16.13.1",
+    "react-app-polyfill": "^2.0.0",
+    "react-dom": "^16.13.1",
+    "react-icons": "^4.1.0",
+    "react-scripts": "4.0.1",
+    "web-vitals": "^0.2.4",
+    "webpack-bundle-tracker": "0.4.3"
```

Comment:
I think some of these aren't necessary for a basic Django/React integration - could you look into paring it down to the bare minimum? Probably something like:
* @?babel*?
* django-react-loader
* react
* react-dom
* webpack-bundle-tracker
* nwb?

For React packages, we should probably make a `sample-react-app` or something (out of scope for this PR).

Also, it might be worth taking some time to discuss what the various babel libraries are doing. If they're doing stuff like polyfill and such, then I agree with keeping them in here (since we'll want those always). But if they're adding additional not-necessarily-important functionality, I'd prefer to leave them off. Here's an example of a `package.json` from another app that is in production for comparison:
```
  "dependencies": {
    "django-react-loader": "^0.1.7",
    "nwb": "^0.24.7",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "webpack-bundle-tracker": "^0.4.3",
    "yarn": "^1.22.10"
  }
```